apps/api/src/app/workflows-v1-deprecated/workflowModuleDeprecated.ts

```diff
@@ -6,15 +6,15 @@ import { MessageTemplateModule } from '../message-template/message-template.modu
 import { SharedModule } from '../shared/shared.module';
 import { NotificationTemplateController } from './notification-template.controller';
 import { USE_CASES } from './usecases';
-import { WorkflowController } from './workflow.controller';
+import { WorkflowControllerDeprecated } from './workflowControllerDeprecated';
 import { PreferencesModule } from '../preferences';
 
 @Module({
   imports: [SharedModule, MessageTemplateModule, ChangeModule, AuthModule, IntegrationModule, PreferencesModule],
-  controllers: [NotificationTemplateController, WorkflowController],
+  controllers: [NotificationTemplateController, WorkflowControllerDeprecated],
   providers: [...USE_CASES],
   exports: [...USE_CASES],
 })
-export class WorkflowModule implements NestModule {
+export class WorkflowModuleDeprecated implements NestModule {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Given that we're introducing V2 now, V3 seems inevitable 😛

Can we rename with the version included?
```suggestion
export class WorkflowModuleV2Deprecated implements NestModule {
```‏